.template/addons/crud/app/views/layouts/application/_nav_bar_top.html.slim

```diff
@@ -0,0 +1,33 @@
+.app__nav_bar_top
+  nav.navbar.navbar-expand-lg.navbar-light.bg-light
+    .container-fluid
+      a.navbar-brand[href="#"]
+        | NavbarTop
```

Comment:
Will it still work without this notation `|`? 💭 
```suggestion
        NavbarTop
```

Reply:
This is using the SLIM template engine. Just adding test means the `<navbartop>` HTML element:

<img width="296" alt="image" src="https://github.com/nimblehq/rails-templates/assets/77609814/90d6af90-abe6-4006-9dbe-c9e8432b7aac">


```suggestion
        div NavbarTop
```

An alternative would be to be explicit about `div`, but I find the `|` more elegant 🙈